src/mon/MgrMonitor.cc

```diff
@@ -44,7 +44,8 @@ const static std::map<uint32_t, std::set<std::string>> always_on_modules = {
       "crash",
       "status",
       "balancer",
-      "devicehealth"
+      "devicehealth",
+      "volumes",
```

Comment:
Since volumes doesn't exist yet at this point, shouldn't this be the last patch in the series?